modules/custom/activenet/src/ActivenetClient.php

```diff
@@ -121,4 +131,11 @@ public function __call($method, $args) {
     throw new ActivenetClientException(sprintf('Method %s not implemented yet.', $method));
   }
 
+  public function getActivityDetail($id){
+    $settings = \Drupal::config('activenet.settings');
```

Comment:
TIme to use Dependency INjection
We have to avoid of using these kind of static calls